### PR TITLE
Taking the pledge

### PR DESCRIPTION
**Pull request by ankitwarbhe**
Added @ankitwarbhe as a contributor

### CODE REVIEW

File: src/data/contributors/ankitwarbhe.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Ankit Warbhe",
+  "skills": ["Python", "Machine Learning", "Java", "Data Science"],
+  "cause": "learn new things and contribute in opensource",
```
Comment (Abhishek-kumar09) [inline src/data/contributors/ankitwarbhe.json:4]: Great! @ankitwarbhe, learning and contributing in open source is your cause for code, mine is same :) Just a very small modification.
image
`I code for learning new things and contributing in opensource` is gramatically more appealing.

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

**Comment by Abhishek-kumar09** [comment]
Your readme is awesome man. creative guy!

File: src/data/contributors/ankitwarbhe.json (hunk 1 above)
Comment (ankitwarbhe) [reply src/data/contributors/ankitwarbhe.json:4]: Thank you for correcting me out man. I appreciate you talking efforts

**Comment by ankitwarbhe** [comment]
> Your readme is awesome man. creative guy!

Thank you feels good

**Review by Abhishek-kumar09 — APPROVED** [review]

**Comment by Abhishek-kumar09** [comment]
Hey @ankitwarbhe , Thanks for your PR. Welcome to the community :1st_place_medal:.

**Comment by ankitwarbhe** [comment]
Thank you @Abhishek-kumar09  for merging the pull request.

**Pull request merged**